diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea7f41e..8a1ea78 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,12 +50,5 @@ add_subdirectory(srec_cat)
 add_subdirectory(srec_cmp)
 add_subdirectory(srec_info)
 
-# Tests & support tools
-enable_testing()
-add_subdirectory(test)
-
-# Documentation & Man Pages
-add_subdirectory(doc)
-
 # Package SRecord
 include(CPack)
diff --git a/etc/configure.cmake b/etc/configure.cmake
index 343a70a..6423953 100644
--- a/etc/configure.cmake
+++ b/etc/configure.cmake
@@ -63,10 +63,6 @@ if (HAVE_GCRYPT_H)
   option(HAVE_GCRY_MD_HD_T "libgcrypt HAVE_GCRY_MD_HD_T" ON)
 endif (HAVE_GCRYPT_H)
 
-# ps2pdf used in building the PDF version of the documentation
-find_program(PS2PDF ps2pdf)
-message(STATUS "PS2PDF ${PS2PDF}")
-
 # cygpath used in helping the build run on Windows
 find_program(CYGPATH cygpath)
 message(STATUS "CYGPATH ${CYGPATH}")
@@ -102,33 +98,3 @@ option(_TANDEM_SOURCE ON)
 
 # Enable general extensions on Solaris.
 option(__EXTENSIONS__ ON)
-
-# Doxygen configuration
-find_package(Doxygen REQUIRED doxygen dot)
-
-set(DOXYGEN_DOT_GRAPH_MAX_NODES 150)
-set(DOXYGEN_ALPHABETICAL_INDEX NO)
-set(DOXYGEN_BUILTIN_STL_SUPPORT YES)
-set(DOXYGEN_CASE_SENSE_NAMES NO)
-set(DOXYGEN_CLASS_DIAGRAMS YES)
-set(DOXYGEN_DISTRIBUTE_GROUP_DOC YES)
-set(DOXYGEN_EXCLUDE bin)
-set(DOXYGEN_EXTRACT_ALL YES)
-set(DOXYGEN_EXTRACT_LOCAL_CLASSES NO)
-set(DOXYGEN_FILE_PATTERNS *.h)
-set(DOXYGEN_GENERATE_TREEVIEW YES)
-set(DOXYGEN_HIDE_FRIEND_COMPOUNDS YES)
-set(DOXYGEN_HIDE_IN_BODY_DOCS YES)
-set(DOXYGEN_HIDE_UNDOC_CLASSES YES)
-set(DOXYGEN_HIDE_UNDOC_MEMBERS YES)
-set(DOXYGEN_JAVADOC_AUTOBRIEF YES)
-set(DOXYGEN_QT_AUTOBRIEF YES)
-set(DOXYGEN_QUIET YES)
-set(DOXYGEN_RECURSIVE YES)
-set(DOXYGEN_REFERENCED_BY_RELATION YES)
-set(DOXYGEN_REFERENCES_RELATION YES)
-set(DOXYGEN_SORT_BY_SCOPE_NAME YES)
-set(DOXYGEN_SORT_MEMBER_DOCS NO)
-set(DOXYGEN_SOURCE_BROWSER YES)
-set(DOXYGEN_STRIP_CODE_COMMENTS NO)
-set(DOXYGEN_GENERATE_HTML YES)
diff --git a/etc/my_install_script.cmake.in b/etc/my_install_script.cmake.in
deleted file mode 100644
index bb6c45e..0000000
--- a/etc/my_install_script.cmake.in
+++ /dev/null
@@ -1,7 +0,0 @@
-# Add the build instructions into the source distribution
-if(CPACK_SOURCE_INSTALLED_DIRECTORIES)
-  file(
-    INSTALL "@PROJECT_BINARY_DIR@/doc/BUILDING.pdf"
-    DESTINATION "${CMAKE_CURRENT_BINARY_DIR}"
-  )
-endif()
diff --git a/etc/packaging.cmake b/etc/packaging.cmake
index ee2a32c..4907a33 100644
--- a/etc/packaging.cmake
+++ b/etc/packaging.cmake
@@ -80,10 +80,6 @@ set(CPACK_NSIS_DISPLAY_NAME "SRecord")
 set(CPACK_NSIS_PACKAGE_NAME "SRecord")
 set(CPACK_NSIS_ENABLE_UNINSTALL_BEFORE_INSTALL YES)
 
-# Helper script to add the BUILDING instructions into the source distribution
-configure_file(etc/my_install_script.cmake.in my_install_script.cmake @ONLY)
-set(CPACK_INSTALL_SCRIPT "${CMAKE_BINARY_DIR}/my_install_script.cmake")
-
 set(CPACK_SOURCE_IGNORE_FILES
   /\\.git/
   \\.gitignore
diff --git a/srecord/CMakeLists.txt b/srecord/CMakeLists.txt
index 930dbf9..28921c5 100644
--- a/srecord/CMakeLists.txt
+++ b/srecord/CMakeLists.txt
@@ -33,17 +33,7 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
         FILES_MATCHING PATTERN "*.h")
 
-# Install the Doxygen output for the website
-install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/html
-        DESTINATION ${CMAKE_INSTALL_DOCDIR}/htdocs/srecord
- 	COMPONENT website)
-
 # Generate config.h to suit local environment
 configure_file(config.h.in config.h @ONLY)
 configure_file(patchlevel.h.in patchlevel.h @ONLY)
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
-
-# Generate doxygen content
-doxygen_add_docs(doxygen
-                 ${LIB_SRECORD_HDR}
-		 ALL)
