diff --git a/etc/packaging.cmake b/etc/packaging.cmake
index 4907a33..4673f37 100644
--- a/etc/packaging.cmake
+++ b/etc/packaging.cmake
@@ -45,12 +45,6 @@ FUNCTION(INSTALL_SRECORD_EXECUTABLE_TARGET target)
 
   install(IMPORTED_RUNTIME_ARTIFACTS ${target}
     RUNTIME_DEPENDENCY_SET ${PROJECT_NAME}_dlls COMPONENT ${target})
-  install(RUNTIME_DEPENDENCY_SET ${PROJECT_NAME}_dlls
-    PRE_EXCLUDE_REGEXES "api-ms-" "ext-ms-"
-    POST_EXCLUDE_REGEXES ".*system32/.*\\.dll"
-    DIRECTORIES ${DLL_SEARCH_DIRS}
-    COMPONENT ${target}
-    )
 ENDFUNCTION()
 
 # Packaging
